WORKSPACE

```diff
@@ -80,6 +80,14 @@ http_archive(
     url = "https://github.com/nodejs/nbytes/archive/refs/tags/v0.1.1.tar.gz",
 )
 
+http_archive(
+    name = "ncrypto",
+    sha256 = "c8284e4719130916cbce340989a59249e605dfbdecd496910d6ed2d1ecaf69ad",
+    strip_prefix = "ncrypto-main",
```

Comment:
Let's make sure we release a new version before merging this PR.

Reply:
Yep, there's likely another PR or two that I'll be adding as I work on a few things. Doing an actual release is one of the necessary next steps.